src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -32,6 +32,10 @@ export type BreadcrumbProps = {
   };
   createURL(value: string | undefined): string;
   refine(value: string | undefined): void;
+  /**
+   * @internal
+   */
+  canRefine?: boolean;
```

Comment:
Seems that `canRefine` property exists somewhere, but is not public. Do you know where it comes from? And why is used?

Reply:
@francoischalifour Component props are always equal to: WidgetOptions + ComponentSpecificProps?

Reply:
On top of my mind I don't know if it's always the case—but component props are for internal use anyway, so no need to over-abstract them.

Reply:
Ok, adding just the can refine.